UnityProject/Assets/Scripts/Items/Bureaucracy/Photocopier.cs

```diff
@@ -7,16 +7,14 @@
 
 namespace Items.Bureaucracy
 {
-	public class Photocopier : NetworkBehaviour, ICheckedInteractable<HandApply>, IServerSpawn
+	public class Photocopier : NetworkBehaviour, ICheckedInteractable<HandApply>, IServerSpawn, IRightClickable
 	{
 		public NetTabType NetTabType;
 		public int trayCapacity;
 
-		[SyncVar]
-		private Internal.Printer printer;
+		[SyncVar] private Internal.Printer printer;
 
-		[SyncVar]
-		private Internal.Scanner scanner;
+		[SyncVar] private Internal.Scanner scanner;
```

Comment:
can we syncvar these objects? I think Mirror only supported basic types and Unity objects

Reply:
It seems to be already working so im not going to change it

Reply:
I didn't say it isn't working. I said it won't be a problem until it becomes one and then we will have to read profiles and rediscover this to solve the issue. Always cache the result of get component when you can.

Reply:
Ah shit, I read the wrong thing

Reply:
I'm glad im not the only one who misreads stuff here lol

Reply:
Whats' your say on this @corp-0 ? is it good to go?

Reply:
We don't want to do this. These objects will be instantiated on every sync, according to Mirror: https://mirror-networking.gitbook.io/docs/guides/data-types (`Classes` bullet point).

Reply:
Didn't realise the code was already syncing those classes. Would've let it slide but good to get it fixed, cheers. So it still works on client/server?

Reply:
I have said this before, this is not my code; it works as it is and I didn't do anything except change the format of where the attribute is.